tests/html-manifest.jsonld

```diff
@@ -275,12 +275,25 @@
   }, {
     "@id": "#tf004",
     "@type": ["jld:PositiveEvaluationTest", "jld:FlattenTest", "jld:HtmlTest"],
-    "name": "Flattens all script elements by default",
+    "name": "Flattens first script element by default",
     "purpose": "Tests embedded JSON-LD in HTML multiple script elements",
     "input": "html/f004-in.html",
     "context": "html/f004-context.jsonld",
     "expect": "html/f004-out.jsonld",
     "option": {"specVersion": "json-ld-1.1", "processorFeature": "HTML Script Extraction"}
```

Comment:
For robustness, I would suggest to also spell out `"extractAllScripts": false` here. But perhaps that would break consistency in the test suite (by not always relying on the implicit defaults)?

Reply:
Yes, explicitly saying with `extractAllScripts` `false` implies that the option test element contain that property; otherwise, it falls back to the default as specified in the API, which is that `extractAllScripts` is `false`.